_includes/pbc/all/install-features/202304.0/install-the-product-feature.md

```diff
@@ -0,0 +1,401 @@
+| ProductDiscontinuedStorage   | spryker/product-discontinued-storage    |
+| ProductImageStorage          | spryker/product-image-storage           |
+| ProductManagement            | spryker/product-management              |
+| ProductPageSearch            | spryker/product-page-search             |
+| ProductQuantityStorage       | spryker/product-quantity-storage        |
+| ProductSearch                | spryker/product-search                  |
+| ProductStorage               | spryker/product-storage                 |
+| ProductSearchConfigStorage   | spryker/product-search-config-storage   |
+
+{% endinfo_block %}
+
+
+### 2) Set up database schema and transfer objects
+
+Run the following commands to apply database changes and generate entity and transfer changes:
+
+```bash
+console transfer:generate
+console propel:install
+console transfer:generate
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following changes have been applied in transfer objects:
+
+| TRANSFER | TYPE | EVENT | PATH |
+| --- | --- | --- | --- |
+| ProductImageFilter | class | created | src/Generated/Shared/Transfer/ProductImageFilterTransfer |
+| ProductConcretePageSearch.images | property | added | src/Generated/Shared/Transfer/ProductConcretePageSearchTransfer |
+
+{% endinfo_block %}
+
+### 3) Add Translations
+
+Append glossary according to your configuration:
+
+**src/data/import/glossary.csv**
+
+```yaml
+quick-order.input.placeholder,Search by SKU or Name,en_US
+quick-order.input.placeholder,Suche per SKU oder Name,de_DE
+product_quick_add_widget.form.quantity,"# Qty",en_US
+product_quick_add_widget.form.quantity,"# Anzahl",de_DE
+quick-order.search.no_results,Item cannot be found,en_US
+quick-order.search.no_results,Das produkt konnte nicht gefunden werden.,de_DE
+product_search_widget.search.no_results,Products were not found.,en_US
+product_search_widget.search.no_results,Products were not found.,de_DE
+```
+
+Run the following console command to import data:
+
+```bash
+console data:import glossary
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the configured data is added to the `spy_glossary` table in the database.
+
+{% endinfo_block %}
+
+### 4) Set up behaviour
+
+1. Enable the following behaviors by registering the plugins:
+
+| PLUGIN                                       | SPECIFICATION                                                                                  | PREREQUISITES | NAMESPACE                                                       |
+|----------------------------------------------|------------------------------------------------------------------------------------------------|---------------|-----------------------------------------------------------------|
+| ProductViewDiscontinuedOptionsExpanderPlugin | Expands the product view super attributes and attribute variant map with discontinued postfix. |               | Spryker/Client/ProductDiscontinuedStorage/Plugin/ProductStorage |
```

Comment:
where is it taken from? specification from a corresponding client method can be used

Reply:
Taken from the plugin specification